src/geometry/basic-geometry.md

```diff
@@ -246,7 +246,7 @@ ftype cross(point2d a, point2d b) {
 
 ### Properties
 
-As for the cross product, it equals to the zero vector iff the vectors $\mathbf a$ and $\mathbf b$ are collinear (they form a common line, i.e. they are parallel).
+As for the cross product, it equals to the zero vector if the vectors $\mathbf a$ and $\mathbf b$ are collinear (they form a common line, i.e. they are parallel).
```

Comment:
**iff** was not a typo. It's a common abbreviation for *if and only if*. See https://en.wikipedia.org/wiki/If_and_only_if